pkg/plugins/platform/utils.go

```diff
@@ -128,32 +128,6 @@ func ComposeFileExists(app App) bool {
 	return true
 }
 
-// Cleanup will clean up ddev apps even if the composer file has been deleted.
-func Cleanup(app App) error {
```

Comment:
if we rely on the `com.ddev.site-name` labels I don't think we can go wrong. If someone has modified their labels the approach would break down, but I don't see any in trying.